sequence_processing_pipeline/Pipeline.py

```diff
@@ -91,21 +96,28 @@ def get_date(run_directory):
 
 
 class Pipeline:
-    sif_header = ['sample_name', 'collection_timestamp', 'elevation', 'empo_1',
-                  'empo_2', 'empo_3', 'env_biome', 'env_feature',
+    _CONTROLS_SIF_SUFFIX = '_blanks.tsv'
+
+    # TODO: replace these with spp_metadata package call based on qiimp2
+    sif_header = [SAMPLE_NAME_KEY, 'collection_timestamp', 'elevation',
+                  'empo_1', 'empo_2', 'empo_3',
+                  'empo_4', 'env_biome', 'env_feature',
                   'env_material', 'env_package', 'geo_loc_name',
-                  'host_subject_id', 'latitude', 'longitude', 'sample_type',
-                  'scientific_name', 'taxon_id', 'description', 'title',
-                  'dna_extracted', 'physical_specimen_location',
-                  'physical_specimen_remaining']
+                  'host_subject_id', 'latitude', 'longitude',
+                  'sample_type', 'scientific_name', 'taxon_id',
+                  'description', 'title', 'dna_extracted',
+                  'physical_specimen_location', 'physical_specimen_remaining']
 
-    sif_defaults = [None, None, 193, 'Control', 'Negative',
+    sif_defaults = [None, None, 193,
```

Comment:
I worry that manually maintaining two parallel lists which are expected to be in index order as being risky. It may be more explicit to model this as a `dict`

Reply:
Agreed that this is not good, but I have not done a little refactor here because it is going to need BIG refacor soon to support katharoseq.  Katharoseq controls will also need to be added into Qiita from the SPP, but neither the current parallel lists approach nor a dictionary approach can gracefully accommodate the differing standard column settings for an ever-growing set of multiple types of controls.  However, qiimp2 can (having been built for that), and I have validated that (in my own environment where qiimp is installed), I can replace all the definitions of the sif fields and defaults with one call to `qiimp.write_extended_metadata_from_df`.  Soon, soon ... :)